### PR TITLE
Fix sync process

### PR DESCRIPTION
**Pull request by fmulero**
Signed-off-by: Fran Mulero <fmulero@vmware.com>

**Description of the change**

`bitnami/containers` repository contains all commits from each container in a plain structure (it doesn't have merge commits) Getting the list of commits in topological order helps to plain the tree ignoring the merge commits.

**Benefits**

Current sync errors with pgbouncer will be fixed

**Possible drawbacks**

Not found

### CODE REVIEW

File: scripts/sync.sh [hunk 1]
```diff
@@ -64,12 +64,12 @@ findCommitsToSync() {
...
-    while IFS='' read -r line; do commits+=("$line"); done < <(git rev-list "${origin_name}/master" -- .)
+    # Get the revision list in topological order ignoring merge commits
+    while IFS='' read -r line; do commits+=("$line"); done < <(git rev-list --topo-order --no-merges "${origin_name}/master" -- .)
     for commit in "${commits[@]}"; do
         if [[ "$commit" != "$last_synced_commit_id" ]] && [[ "$max" -gt "0" ]]; then
-            actual_commit_id="$(plainCommit "${commit}")"
-            if [[ "$commit" != "$SKIP_COMMIT_ID" ]] && [[ ! $commits_to_sync =~ (^|[[:space:]])$actual_commit_id($|[[:space:]]) ]]; then
-              commits_to_sync="${actual_commit_id} ${commits_to_sync}"
+            if [[ "$commit" != "$SKIP_COMMIT_ID" ]]; then
+              commits_to_sync="${commit} ${commits_to_sync}"
```
Comment (alemorcuq) [inline scripts/sync.sh:72]: Why this change?

Comment (fmulero) [reply scripts/sync.sh:72]: Now we don't need to resolve merge commits, so `plainCommit` is not needed and there is no risk to add twice the same commit

**Review by alemorcuq — APPROVED** [review]
